test/export/test_export.py

```diff
@@ -2985,6 +2985,15 @@ def forward(self, l_q_, l_k_, l_v_):
     getitem = _scaled_dot_product_flash_attention[0];  _scaled_dot_product_flash_attention = None
     return (getitem,)""")
 
+    def test_int_list_output(self):
```

Comment:
Why the test is for list_output instead of user inputs? I thought the code change is for inputs

Reply:
original fix was for the handling outputs. and then i decided to fix the inputs the same way so that we know clearly the types of inputs allowed...but that caused a test failure.

Reply:
ohh i understand... i fixed the merge conflict badly

Reply:
ok, probably add a test for input as well for symetry?